alvr/server_openvr/src/tracking.rs

```diff
@@ -262,3 +262,44 @@ pub fn to_openvr_ffi_hand_skeleton(
 
     to_ffi_skeleton(&skeleton)
 }
+
+pub fn offset_motion(config: &HeadsetConfig, device_id: u64, motion: &mut DeviceMotion) {
+    let (t, r) = if let Switch::Enabled(controllers) = &config.controllers {
+        (
+            controllers.left_controller_position_offset,
+            controllers.left_controller_rotation_offset,
+        )
+    } else {
+        return;
+    };
+
+    let pose_offset = if device_id == *HAND_LEFT_ID {
+        Pose {
+            orientation: Quat::from_euler(
+                EulerRot::XYZ,
+                r[0] * DEG_TO_RAD,
+                r[1] * DEG_TO_RAD,
+                r[2] * DEG_TO_RAD,
+            ),
+            position: Vec3::new(t[0], t[1], t[2]),
+        }
+    } else if device_id == *HAND_RIGHT_ID {
+        Pose {
+            orientation: Quat::from_euler(
+                EulerRot::XYZ,
+                r[0] * DEG_TO_RAD,
+                -r[1] * DEG_TO_RAD,
+                -r[2] * DEG_TO_RAD,
+            ),
+            position: Vec3::new(-t[0], t[1], t[2]),
+        }
+    } else {
+        return;
```

Comment:
Honestly I'd prefer an `unreachable` here, because if something else ever gets input this should trip up because it not doing anything would prolly be unexpected.
Or better yet just get a bool for the hand as an input (or maybe we should make a global enum for this, idk).

Reply:
Or if you actually intend to extend the function later maybe slap a `todo!()` there.

Reply:
Probably not. This function is just to fix the offsets caused by SteamVR, which they didn't fix for legacy reasons. Other trackers are not affected